hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/feature/index/testHoodieBackedTableMetadataIndexLookup.scala

```diff
@@ -54,7 +54,29 @@ abstract class HoodieBackedTableMetadataIndexLookupTestBase extends HoodieSparkS
   protected var hoodieBackedTableMetadata: HoodieBackedTableMetadata = _
   protected var testData: Seq[Seq[Any]] = _
   protected var tmpDir: File = _
-
+  private val createTableStmtProvider = () =>
```

Comment:
nit: try to avoid abbreviations
```suggestion
  private val createTableStatementProvider = () =>
```

Reply:
done